tests/libs/test_github.py

```diff
@@ -33,6 +33,84 @@ class TestGithub(unittest.TestCase):
         },
     ]
 
+    _sample_response_branches = {
```

Comment:
Maybe best to make this as small as we need, a couple of entries with name commit and sha should probably do